### DISCUSSION TITLE
new icons

### DISCUSSION THREAD

**Discussion by gajamaps** [uikit/uikit#4582] [General]
hi, i adore uikit but i found the icons are a bit limited. so i created some additional ones matching the same style. hope you can integrate them. There are no license issues with them, cause i created them myself. i would love to contribute them to uikit.
[icons.zip](https://github.com/uikit/uikit/files/8193968/icons.zip)

**Comment by gajamaps** [comment]
how can i contribute the icons? anyone responsible for it?

**Reply by AlekVolsk** [reply]
accept: no one will add anything, this is not accepted here

**Reply by AlekVolsk** [reply]
if you need these icons - add them in your project, individually: there is a corresponding interface for this

**Reply by gajamaps** [reply]
i already added them to my project. thats not my concern. i wanted to share them with the community.

**Reply by gajamaps** [reply]
accept: its a contribution, not a request

**Reply by AlekVolsk** [reply]
then make your contribution as git is supposed to: make a pull request
although who needs it - everyone has already downloaded and silently said thanks)
